driver/src/main/java/org/neo4j/driver/internal/metrics/spi/PoolStatus.java

```diff
@@ -20,5 +20,18 @@
 
 public enum PoolStatus
 {
-    Open, Closed
+    Open( 0 ),
+    Closed( 1 );
+
+    private final int value;
```

Comment:
Can this field be removed? It's really up to the users how to interpret the returned status

Reply:
But I kind of do not want to have a if-else just for this conversion when publishing the result.

```metrics.add( new Metric( path( serverName, "poolStatus" ), metric.poolStatus().value(), date ) );```

vs.

```
PoolStatus staus = metric.poolStatus()
int statusNum = -1; 
if( status == PoolStatus.OPEN)
{
    statusNum = 0;
}
else if( status == PoolStatus.CLOSED)
{
   statusNum = 1;
}
metrics.add( new Metric( path( serverName, "poolStatus"), statusNum, date ) );
```

I feel for a metrics collector, it does not matter what value you assigned to each PoolStatus.
It is more important to record all possible PoolStatus. I mean it would be good to know the real value, but it will not be a big issue if a user does not know as long as they see they are different. So I thought it would be helpful to have a simple `value` to use directly.